web/server/watcher.py

```diff
@@ -27,8 +27,4 @@ async def watch_project(queue: asyncio.Queue) -> None:
         except Exception:
             logger.exception("Error loading context")
         else:
-            queue.put_nowait(
-                Event(
-                    event="models", data=json.dumps([model.dict() for model in get_models(context)])
-                )
-            )
+            queue.put_nowait(Event(event="models", data=jsonable_encoder(get_models(context))))
```

Comment:
it seems we still need to wrap in `json.dumps` to send valid json
`json.dumps(jsonable_encoder(get_models(context))))`